aiida/tools/importexport/dbimport/__init__.py

```diff
@@ -64,13 +64,15 @@ def import_data(in_path, group=None, silent=False, **kwargs):
     from aiida.backends import BACKEND_DJANGO, BACKEND_SQLA
     from aiida.tools.importexport.common.exceptions import ArchiveImportError
 
-    if configuration.PROFILE.database_backend == BACKEND_SQLA:
+    backend = configuration.PROFILE.database_backend
+
+    if backend == BACKEND_SQLA:
         from aiida.tools.importexport.dbimport.backends.sqla import import_data_sqla
         return import_data_sqla(in_path, group=group, silent=silent, **kwargs)
 
-    if configuration.PROFILE.database_backend == BACKEND_DJANGO:
+    if backend == BACKEND_DJANGO:
```

Comment:
```suggestion
    elif backend == BACKEND_DJANGO:
```

Reply:
No need, since the previous statement is a `return`.

Reply:
It's not super important of course, but my change saves the evaluation of the second if condition if the first one is met.
It is always the right thing to do if you have multiple mutually exclusive options.

Reply:
> It's not super important of course, but my change saves the evaluation of the second if condition if the first one is met.

Are you sure?
I think Python reads from the top down, so no matter what, the first `if`-statement will always be evaluated first, and if True, the block will be executed, and first _then_ the second `if`-statement will be evaluated, however, it never will, since the first `if`-block will have returned out of the evaluation process.. right?